docker/puppetdb/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM ubuntu:16.04
+
+ARG vcs_ref
+ARG build_date
+ENV PUPPETDB_VERSION="5.2.2"
+ENV PUPPET_AGENT_VERSION="5.5.1"
+ENV DUMB_INIT_VERSION="1.2.1"
+ENV UBUNTU_CODENAME="xenial"
+ENV PUPPETDB_DATABASE_CONNECTION="//postgres:5432/puppetdb"
+ENV PUPPETDB_USER=puppetdb
+ENV PUPPETDB_PASSWORD=puppetdb
+ENV PUPPETDB_JAVA_ARGS="-Djava.net.preferIPv4Stack=true -Xms256m -Xmx256m"
+ENV PATH=/opt/puppetlabs/server/bin:/opt/puppetlabs/puppet/bin:/opt/puppetlabs/bin:$PATH
+
+LABEL org.label-schema.maintainer="Puppet Release Team <release@puppet.com>" \
+      org.label-schema.vendor="Puppet" \
+      org.label-schema.url="https://github.com/puppetlabs/puppetdb" \
+      org.label-schema.name="PuppetDB" \
+      org.label-schema.license="Apache-2.0" \
+      org.label-schema.version=$PUPPETDB_VERSION \
+      org.label-schema.vcs-url="https://github.com/puppetlabs/puppetdb" \
+      org.label-schema.vcs-ref=$vcs_ref \
+      org.label-schema.build-date=$build_date \
+      org.label-schema.schema-version="1.0" \
+      org.label-schema.dockerfile="/Dockerfile"
```

Comment:
Suggest quoting for vcs_ref, PUPPETDB_VERSION, build_date, etc. here too unless the LABEL command does something special, just to be safe wrt accidental injection, etc.